### PR TITLE
fix(gateway/oas-validation): add a proper breaking change entry

### PR DESCRIPTION
**Pull request by vm-001**
### Description

Add breaking change entries to 3.4.4.1, 3.5.0.2, and 3.6.0.0.

 

### Testing instructions

Preview link: 

### Checklist 

- [ ] Review label added 
- [ ] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

For example, if this change is for an upcoming 3.6 release, enclose your content in `{% if_version gte:3.6.x %} <content> {% endif_version %}` tags (or `if_plugin_version` tags for plugins). 

Use any of the following keys:
* `gte:<version>` - greater than or equal to a specific version
* `lte:<version>` - less than or equal to a specific version
* `eq:<version>` - exactly equal to a specific version

You can do the same for older versions.

[FTI-5747]

[FTI-5747]: https://konghq.atlassian.net/browse/FTI-5747?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lena-larionova** [comment]
This breaking change isn't in any of these releases, is it? I see your PR was just merged two days ago, so it can't be in an existing patch: https://github.com/Kong/kong-ee/pull/8255

**Comment by vm-001** [comment]
@lena-larionova The behavior changes actually introduced by PR https://github.com/Kong/kong-ee/pull/7475. The https://github.com/Kong/kong-ee/pull/8255 just updates the field description to be used by the doc site. This PR just add the missing documentation.

File: app/gateway/changelog.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ configuration field to construct the request path when requesting the Azure API.
 
 * [**SAML**](/hub/kong-inc/saml) (`saml`)
   * Adjusted the priority of the SAML plugin to 1010 to correct the integration between the SAML plugin and other consumer-based plugins.
+* [**OAS Validation**](/hub/kong-inc/oas-validation) (`oas-validation`): The field `api_spec` no longer accepts URl-encoded content. Please use the raw content instead.
```
Comment (lena-larionova) [inline app/gateway/changelog.md:65]:
```suggestion
* [**OAS Validation**](/hub/kong-inc/oas-validation) (`oas-validation`): 
  * The field `api_spec` no longer accepts URL-encoded content. Use raw content instead.
```

File: app/gateway/changelog.md [hunk 2]
```diff
@@ -485,6 +486,7 @@ both in the Admin API and in Kong Manager.
 #### Plugins
 * [**SAML**](/hub/kong-inc/saml) (`saml`)
   * Adjusted the priority of the SAML plugin to 1010 to correct the integration between the SAML plugin and other consumer-based plugins.
+* [**OAS Validation**](/hub/kong-inc/oas-validation) (`oas-validation`): The field `api_spec` no longer accepts URl-encoded content. Please use the raw content instead.
```
Comment (lena-larionova) [inline app/gateway/changelog.md:489]:
```suggestion
* [**OAS Validation**](/hub/kong-inc/oas-validation) (`oas-validation`): 
  * The field `api_spec` no longer accepts URL-encoded content. Use raw content instead.
```

File: app/gateway/changelog.md [hunk 3]
```diff
@@ -940,6 +942,7 @@ and Kong Gateway will try to control the concurrency when exporting the config.
 ### Breaking Changes
 #### Plugins
 * [**SAML**](/hub/kong-inc/saml) (`saml`): Adjusted the priority of the SAML plugin to 1010 to correct the integration between the SAML plugin and other consumer-based plugins.
+* [**OAS Validation**](/hub/kong-inc/oas-validation) (`oas-validation`): The field `api_spec` no longer accepts URl-encoded content. Please use the raw content instead.
```
Comment (lena-larionova) [inline app/gateway/changelog.md:945]:
```suggestion
* [**OAS Validation**](/hub/kong-inc/oas-validation) (`oas-validation`): 
  * The field `api_spec` no longer accepts URL-encoded content. Use raw content instead.
```

**Comment by lena-larionova** [comment]
@vm-001 double-checking, is it supposed to be "URI-encoded" or "URL-encoded"?

**Pull request closed without merging**